### ISSUE TITLE
Apache 2.0 vs. MIT License

### ISSUE THREAD

**Issue by axunonb** [StefH/ReflectionBridge#4]
Thanks for sharing ReflectionBridge. I'm contributing to a project which is licensed under The MIT License, where ReflectionBridge could be really helpful. Not sure whether the project owner would accept parts under Apache 2.0. MIT is not an option for ReflectionBridge, is it?

**Comment by StefH** [comment]
Changed to MIT ;-)

**Comment by axunonb** [comment]
Thanks, that's great. In the meantime I succeeded in porting [YAXLib](https://github.com/sinairv/YAXLib) to .Net Core, also thanks to ReflectionBridge and the idea behind. I have attached my modified version, where I also fixed several issues, specifically with ```InvokeMethod```. Feel free to include it into ReflectionBridge if you think the updated/added parts makes sense to you.
[ReflectionBridgeExtensions.zip](https://github.com/StefH/ReflectionBridge/files/768932/ReflectionBridgeExtensions.zip)
